### PR TITLE
Routing node cleanup

### PR DESCRIPTION
**Pull request by Arcaratus**
-Made the code neater
-Cleaned up the texture/button placement

### CODE REVIEW

File: src/main/java/WayofTime/bloodmagic/client/gui/GuiItemRoutingNode.java [hunk 1]
```diff
@@ -54,15 +49,18 @@ private int getCurrentActiveSlotPriority()
...
-        this.buttonList.add(this.upButton = new GuiButton(1, (this.width - this.xSize) / 2 + 133, (this.height - this.ySize) / 2 + 14, 18, 18, "U"));
-        this.buttonList.add(this.northButton = new GuiButton(2, (this.width - this.xSize) / 2 + 151, (this.height - this.ySize) / 2 + 14, 18, 18, "N"));
-        this.buttonList.add(this.southButton = new GuiButton(3, (this.width - this.xSize) / 2 + 151, (this.height - this.ySize) / 2 + 50, 18, 18, "S"));
-        this.buttonList.add(this.westButton = new GuiButton(4, (this.width - this.xSize) / 2 + 133, (this.height - this.ySize) / 2 + 32, 18, 18, "W"));
-        this.buttonList.add(this.eastButton = new GuiButton(5, (this.width - this.xSize) / 2 + 151, (this.height - this.ySize) / 2 + 32, 18, 18, "E"));
-        this.buttonList.add(this.incrementButton = new GuiButton(6, (this.width - this.xSize) / 2 + 97, (this.height - this.ySize) / 2 + 14, 18, 17, "^"));
-        this.buttonList.add(this.decrementButton = new GuiButton(7, (this.width - this.xSize) / 2 + 97, (this.height - this.ySize) / 2 + 50, 18, 17, "v"));
+        this.buttonList.add(this.downButton = new GuiButton(0, left + 133, top + 50, 18, 17, "D"));
```
Comment (WayofTime) [inline src/main/java/WayofTime/bloodmagic/client/gui/GuiItemRoutingNode.java:56]: Why did you change these to 17 from 18?

File: src/main/java/WayofTime/bloodmagic/client/gui/GuiItemRoutingNode.java [hunk 2]
```diff
@@ -20,17 +20,12 @@
...
 {
-    private GuiButton downButton;
-    private GuiButton upButton;
-    private GuiButton northButton;
-    private GuiButton southButton;
-    private GuiButton westButton;
-    private GuiButton eastButton;
-    private GuiButton incrementButton;
-    private GuiButton decrementButton;
+    private GuiButton downButton, upButton, northButton, southButton, westButton, eastButton, incrementButton, decrementButton;
```
Comment (TehNut) [inline src/main/java/WayofTime/bloodmagic/client/gui/GuiItemRoutingNode.java:23]: Moving fields onto one line doesn't fit the formatting of anywhere else in the project.

File: src/main/java/WayofTime/bloodmagic/client/gui/GuiItemRoutingNode.java (hunk 1 above)
Comment (Arcaratus) [inline src/main/java/WayofTime/bloodmagic/client/gui/GuiItemRoutingNode.java:56]: Fixes the texture mismatch-y bug
